Sources/MFBActionBinding.h

```diff
@@ -0,0 +1,72 @@
+//
+//  MFBActionBinding.h
+//  Pods
+//
+//  Created by Nickolay Tarbayev on 28.07.2017.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "NSObject+MFBBindingAssertion.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MFBActionBinding : NSObject
```

Comment:
I find it a bit too hacky. UIControl has a long inheritance chain not worth to deal with. Also we cannot limit UIControlEvents list available from IB.